doc/modules/ROOT/pages/projects.adoc

```diff
@@ -431,7 +431,7 @@ Each helper means `projectile-related-files-fn-helper-name` function.
 
 (projectile-register-project-type
    ;; ...
-   :related-files-fn #'my/related-files)
```

Comment:
This definitely doesn't seem like a typo to me.

Reply:
`my/related-files` is a list here, so I find using `:related-files-fn #'my/related-files` results in an `unsupported value type of related-files-fn`, or am I missing something here?

Reply:
That's supposed to be a function, not a list.

Reply:
But `related-files'fn` can accept a list as well:
https://github.com/bbatsov/projectile/blob/793abfead25a7ef92ad3a443ad3067a82f4a2fb9/projectile.el#L2417
 which is what this example is showing?

Reply:
Oh, my bad. I should have expanded the diff.